.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Responsible Disclosure Policy
+
+At the IOTA Foundation, we consider the security of our systems a top priority. Nevertheless, no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
```

Comment:
Perhaps this is a better, less repetitive phrasing:

`However; despite our best efforts there may still be vulnerabilities present.`

Reply:
Done